cypress/e2e/api.cy.js

```diff
@@ -0,0 +1,14 @@
+/// <reference types = "cypress" />
+
```

Comment:
Evite excesso de linhas em branco.

# Por que tal prática fere um bom design de testes?

Em vez de ajudar na legibilidade do código, tal prática piora a mesma, visto que torna os arquivos maiores do que o necessário.

## Sugestão com um melhor design

Não deixe linhas em branco no início do arquivo, nem mais de uma linha entre diferentes “coisas” (ex.: blocos `it`, `beforeEach`, `context`, etc.). Uma linha em branco já é o suficiente para deixar a suíte de testes legível.

Veja um exemplo.

```js
describe('EngageSphere', () => {
  beforeEach(() => {
    cy.visit('/')
  })

  it('shows the default greeting (i.e., Hi, there...)', () => {
    // Test implementation here
  })

  it('shows the customized greeting (i.e., Hi, Joe...)', () => {
    // Test implementation here
  })

  // Outros casos de teste aqui...
})
```

Reply:
O mesmo vale para outros locais onde tal padrão se repita.